src/utils/tracked-controls.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Enumerate gamepads and apply callback function to each.
+ *
+ * @param {object} callback - callback function that takes gamepad as argument.
+ * @param {object} idPrefix - prefix to match in gamepad id, if any.
+ */
+module.exports.enumerateGamepads = function (callback, idPrefix) {
+  var gamepads = navigator.getGamepads && navigator.getGamepads();
```

Comment:
I would call this `getGamepadsByPrefix` and make it just return an array. If the caller wants to do something with each gamepad it can just do `getGamepadsByPrefix(prefix).forEach(...)`